extensions/assets-manager/AssetsManagerEx.cpp

```diff
@@ -494,7 +530,12 @@ void AssetsManagerEx::downloadManifest()
     if (_updateState != State::PREDOWNLOAD_MANIFEST)
         return;
 
-    std::string manifestUrl = _localManifest->getManifestFileUrl();
+    std::string manifestUrl;
+    if (_remoteManifest->isVersionLoaded()) {
+        manifestUrl = _remoteManifest->getManifestFileUrl();
+    } else {
+        manifestUrl = _localManifest->getManifestFileUrl();
+    }
```

Comment:
Let me explain why we are making this change.

Each member of our team has their own set of assets for development. For example, we separate them by url in this format:

> https://host/<user-name>/xxx/xxx/xxx.png

And we have a master branch of assets 

> https://host/master/xxx/xxx/xxx.png

In the development build, we can switch between assets branches by making AssetsManagerEx update via a different url. But the original implementation won't use the url in the newly downloaded version file, making this switch impossible.